src/geneus/geneus_main.py

```diff
@@ -31,7 +31,6 @@
 # POSSIBILITY OF SUCH DAMAGE.
 #
 
-from __future__ import print_function
```

Comment:
This line can't be dropped if Python 2 should continue to work. And since this branch is used by ROS Kinetic and Melodic I think that is the case.